lib/screens/tabs_screen.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:meals_app/screens/categories_screen.dart';
+import 'package:meals_app/screens/favorites_screen.dart';
+
+class TabsScreen extends StatefulWidget {
+  @override
+  _TabsScreenState createState() => _TabsScreenState();
+}
+
+class _TabsScreenState extends State<TabsScreen> {
+  final List<Map<String, Object>> _pages = [
+    {
+      'page': CategoriesScreen(),
+      'title': 'Categories',
+    },
+    {
+      'page': FavoritesScreen(),
+      'title': 'Your favorite',
+    }
+  ];
+
+  int _selectedPageIndex = 0;
+
+  void _selectPage(int index) {
+    setState(() {
+      _selectedPageIndex = index;
+    });
+  }
```

Comment:
this can be simplified, for example:

`void _selectPage(int index) => setState(() => _selectedPageIndex = index);`

Reply:
@MapleSyyrup why resolved when you haven't simplified this one?